src/components/Map.js

```diff
@@ -1,22 +1,46 @@
 import React from 'react';
 import mapboxgl from 'mapbox-gl';
 import MapboxGeocoder from '@mapbox/mapbox-gl-geocoder';
-import './index.css';
-import { getCategories, getResourceObject } from './map-services.js';
-import Menu from "./components/Menu/Menu";
+import '../index.css';
+import Menu from "./Menu/Menu";
+import {
+    setPopUp
+} from "./PopUp.js";
 
 
 mapboxgl.accessToken = 'pk.eyJ1IjoicmVmdWdlZXN3ZWxjb21lIiwiYSI6ImNqZ2ZkbDFiODQzZmgyd3JuNTVrd3JxbnAifQ.UY8Y52GQKwtVBXH2ssbvgw';
 
 
 class Map extends React.Component {
-    constructor(props){
+    constructor(props) {
         super(props);
         this.state = {
-            providers: [],
-            resources: []
+            providers: []
         }
-   }
+        this.map = null;
+        this.mapContainer = React.createRef();
+        this.handleProviderClick = this.handleProviderClick.bind(this);
+    }
+
+    handleProviderClick(provider) {
```

Comment:
@sashamaryl this doesn't seem to work for me (when I click on a provider in the dropdown, the URL changes to `#target` and nothing else happens), but it didn't work before either, so maybe we can get this working in a subsequent PR?